ckanext/geocat/harvester.py

```diff
@@ -256,6 +311,107 @@ def import_stage(self, harvest_object):  # noqa
             )
             return False
 
+    def _create_new_context(self):
+        # get the site user
+        site_user = tk.get_action('get_site_user')(
+            {'model': model, 'ignore_auth': True}, {})
+        context = {
+            'model': model,
+            'session': Session,
+            'user': site_user['name'],
+        }
+        return context
+
+    def _get_existing_package_names(self, harvest_job):
+        context = self._create_new_context()
+        n = 2
```

Comment:
n should be 500, I think you copied that from what I had in ckanext-stadtzh-harvester and it was wrong there for a while. n=2 was just a serving size for testing, so there would be more than one page, but that size is too small for usage in production. At OGDZH we agreed on n=500 as a reasonable size that goes well with solr.

Reply:
fixed! thank you for catching that :D